examples/volumes/vsphere/vsphere-volume-sc-vsancapabilities.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    hostFailuresToTolerate: "2"
```

Comment:
In this case you were saying that the vsphere cloud provider would create the disk on the global datastore which may not be VSAN and could fail. This may not be expected by the user. The user's expectation would instead be that the volume is created on a VSAN datastore selected by the system.